protocol.md

```diff
@@ -1,58 +1,70 @@
 Magic Crystal
 =============
 
-HOWTO use magic-wormhole as a transport mechannism for Dark Crystal
-shards.
+_SB: Is this doc supposed to be a how-to guide for a (tech-savvy) user, or more a specification for a programmer to build an app?_
+
+This document provides a protocol and examples for using [Magic Wormhole](https://magic-wormhole.readthedocs.io/en/latest/welcome.html) as a transport mechanism for [secret sharing](https://darkcrystal.pw/what-is-secret-sharing/) with [Dark Crystal](https://darkcrystal.pw/) and the Public Key Infrastructure (PKI) of your choice.
+
+_SB: Add TOC when doc is ready_
+
+Introduction
+-------------
+
+[Dark Crystal](https://darkcrystal.pw/) facilitates storing secrets (i.e., sensitive data such as access credentials or cryptographic keys) between trusted peers, rather than relying on individual responsibility or 3rd party providers. Dark Crystal is *transport agnostic*, meaning that in can be used in combination with any transfer/communication mechanisms. 
+
+The transfer mechanism used in this document is [Magic Wormhole](https://magic-wormhole.readthedocs.io/en/latest/welcome.html), a tool for transporting arbitrary-sized files or directories between two computers. Magic Wormhole relies on the assumption that the two humans in charge of the two computers can communicate with each other either in person, on the phone, or via some messaging service.
+
+In addition, some PKI (Public Key Infrastructure) is needed to carry out the protocol described here, and all recipients and sender already need to have public keys for each other. We use [OpenPGP](https://www.openpgp.org/) as an example in this document.
 
```

Comment:
Speak to the value add and purpose. What do users get from this?  Motivational sentences 
value add: soft version of identity due to PKI usage .  
trusted peers is a big part of this so we should keep that. I have stuff to back up and a sufficiently big enough group of my peers can get it back . Example: using exiting PKIs . something you would want to back up. example passcode for someone's encrypted file system
Assume dark crystal is already being used : you decided you have already want to use Dark crystal but your sw doesnt have an immediate way to talk to peers and to send shards to custodians